index.html

```diff
@@ -426,7 +424,11 @@ <h2>kid</h2>
       If <code>kid</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>,
       a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
       to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
```

Comment:
to obtain a JWK to use in the -> to obtain a key identifier to use to select the correct key for use in the

Reply:
This is what @selfissued suggested --
```suggestion
      to obtain a key identifier to use to select the correct key for use in the 
```
-- but as I understand it, `kid` *is* a `key identifier`, so...

Reply:
... might this might be more accurate? It's still not clear to me how the `kid` gets you to the key it identifies, nor where that key is found, so there's the `__` bracketed bit below --
```suggestion
      as a key identifier, enabling selection of the correct key __from the ???__ for use in the
```

Reply:
```suggestion
      as a hint indicating which key was used to secure the verifiable credential when performing a
```
copying from this section https://www.rfc-editor.org/rfc/rfc7515#section-4.1.4 is probably the best

Reply:
@TallTed there is a kid in each JWK in a JWK set (which is obtained either from a DID Doc, .well-known path etc.) and kid in the header is supposed to match with one of those kids in a JWK.